table/table.go

```diff
@@ -145,6 +145,12 @@ type Table interface {
 	Type() Type
 }
 
+// PartitionTable is a Table, and it implements GetPartition() method.
```

Comment:
The comment is not clear.

Reply:
You should explain what is GetPartition()

Reply:
So, any suggestion?